packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/ForkliftControllerDetailsTab.tsx

```diff
@@ -18,13 +20,31 @@ interface ForkliftControllerDetailsTabProps extends RouteComponentProps {
 export const ForkliftControllerDetailsTab: React.FC<ForkliftControllerDetailsTabProps> = ({
   obj,
 }) => {
+  const [userSettings, hideWelcomeCard, setHideWelcomeCard] = useWelcomeStateByUserSettings();
+
+  // Set and use context data for the overview page state
+  const { setData } = useCreateOverviewStateContext();
+  const { data: { hideWelcomeCardByContext } = {} } = useCreateOverviewStateContext();
+
+  const isDisplayingWelcomeCard = () =>
```

Comment:
hmm, we use the words 'show' and 'hide' all over the place, `display`  is a new word, can you continue to use 'show' and 'hide' instead of display ?